components/brave_perf_predictor/browser/perf_predictor_tab_helper.cc

```diff
@@ -23,10 +25,10 @@ content::WebContents* GetWebContents(int render_process_id,
                                      int render_frame_id,
                                      int frame_tree_node_id) {
   content::WebContents* web_contents =
-      content::WebContents::FromFrameTreeNodeId(frame_tree_node_id);
```

Comment:
@deeppandya would you mind restoring proper formatting as per Chromium code style? You can set up `clang-format` to automatically format code in your IDE